Cargo.toml

```diff
@@ -34,12 +33,12 @@ travis-ci = { repository = "dalek-cryptography/x25519-dalek", branch = "master"}
 features = ["nightly", "reusable_secrets", "serde"]
 
 [dependencies]
-curve25519-dalek = { version = "3", default-features = false }
-rand_core = { version = "0.5", default-features = false }
+curve25519-dalek = { version = "4.0.0-pre.2", default-features = false }
+rand_core = { version = "0.6", default-features = false }
 # `serde` is renamed to `our_serde` in order to avoid a name collision between
 # importing the serde dependency and enabling the curve25519-dalek/serde feature
 our_serde = { package = "serde", version = "1", default-features = false, optional = true, features = ["derive"] }
-zeroize = { version = "=1.3", default-features = false, features = ["zeroize_derive"] }
+zeroize = { version = "1", default-features = false, features = ["zeroize_derive"] }
```

Comment:
This patch would unblock the coexistence of this crate with [`elliptic-curve`](https://crates.io/crates/elliptic-curve). What would it take to get it released?

Reply:
It needs to be updated to use https://github.com/dalek-cryptography/curve25519-dalek/issues/405 once that's released